tests/search/test_solr.py

```diff
@@ -10,6 +10,7 @@
 
 
 class BaseSolrTest(unittest2.TestCase):
+
```

Comment:
I'm not picky about this, and I don't think pep8 takes a stance on it, but I do like consistency. The existing style in this module and others is to not use a blank line after a class definition, so I think we should stick with that convention.